README.rst

```diff
@@ -80,4 +80,4 @@ Next Steps
    APIs that we cover.
 
 .. _Cloud Billing API Product documentation:  https://cloud.google.com/billing
-.. _repository’s main README: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
\ No newline at end of file
+.. _repository’s main README: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
```

Comment:
This link still seems to be the same as before? (points to the old repo) Is this correct?

Reply:
Fixed!